src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -75,10 +77,12 @@
 short-summary: Revoke a resource's read access to a managed disk.
 examples:
   - name: Revoke a resource's read access to a managed disk. (autogenerated)
-    text: az disk revoke-access --ids $id
+    text: |
+        az disk revoke-access --ids $id
     crafted: true
   - name: Revoke a resource's read access to a managed disk. (autogenerated)
```

Comment:
Same example names

Reply:
I don't think this is duplicated.

Reply:
The are not duplicate examples. But maybe the names can be more descriptive to distinguish the two examples.